### ISSUE TITLE
Add labels-to-remove-when-stale option

### ISSUE THREAD

**Issue by arnodirlam** [actions/stale#770]
## The problem

We have a workflow where a `deploy` label triggers the deployment of a dedicated test environment on AWS via Terraform (like Heroku's "review apps"). When it's removed, the test environment is destroyed. We'd like to keep this flow, but automatically removed the label when the PR becomes **stale**. Right now, there's only an option to add/remove labels when a PR becomes **unstale**.

## The solution

Please add an option `labels-to-remove-when-stale`.

Basically like the existing [labels-to-remove-when-unstale](https://github.com/actions/stale#labels-to-remove-when-unstale) but triggering when the issue/PR becomes stale (not when it becomes unstale).

Thank you very much for your great work! 🙏

**Comment by mulhotavares** [comment]
That would be interesting.
When a PR comes stale, there are labels that we want to remove automatically.

**Comment by ge65cer** [comment]
+1

**Comment by Lasim** [comment]
+1

**Comment by Kisanja998** [comment]
Love spells, Marriage spells - Bring lost lover back
Stop lover from cheating, be in control of the love in the house. Fix your troubled marriage. Tried many try me last, 35 years of experience. Stop or make a Divorce. Fix a Broken Marriage. Fall in Love & Commitment. Stop cheating.
Love spells, Marriage spells - Bring lost lover back Stop lover from cheating, be in control of the love in the house. Fix your troubled marriage. Tried many try me last, 35 years of experience. Stop or make a Divorce. Stop cheating. Fall in Love & Commitment. Fix a Broken Marriage.
Contact Me
https://chiefaberash.com/
Get in Touch Get instant help for your love life now, with instant and guaranteed results.
Contact +27 63 5340727 Email info@chiefaberash.com

**Comment by panticmilos** [comment]
hi @arnodirlam, we have released a new major [version](https://github.com/actions/stale/releases/tag/v8.0.0) of the action that is including your feature request.

Could you confirm it is working as expected?
